### PR TITLE
refactor(helper/testing): remove some any of helper/testing

### PR DESCRIPTION
**Pull request by EdamAme-x**
### The author should do the following, if applicable

- [ ] Add tests
- [x] Run tests
- [x] `bun run format:fix && bun run lint:fix` to format the code

### CODE REVIEW

File: src/helper/testing/index.ts [hunk 1]
```diff
@@ -7,13 +7,13 @@ import { hc } from '../../client'
 import type { Client } from '../../client/types'
 import type { ExecutionContext } from '../../context'
 import type { Hono } from '../../hono'
+import type { Env, Schema } from '../../types'
```
Comment (yusukebe) [inline src/helper/testing/index.ts:10]: Can you remove `Env` to avoid the linter warning?

Comment (EdamAme-x) [reply src/helper/testing/index.ts:10]: Sorry, I forgot it. wait a moment...

**Comment by EdamAme-x** [comment]
@yusukebe Thanks for reviews.
Ready

**Review by yusukebe — APPROVED** [review]
LGTM!

**Pull request merged**